wrappers/kn/src/nativeMain/kotlin/zxingcpp/ReaderOptions.kt

```diff
@@ -0,0 +1,99 @@
+package zxingcpp
```

Comment:
please move the contents of this file and also the one function in `zxingUitls.kt` into the `BarcodeReader.kt`

Reply:
done https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/4b1197deb6f4e89ecdf4821590ab7995c6ea97b8